Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -427,6 +448,9 @@ extension SourceKitServer {
           log("expected null or int for 'maxResults'; got \(invalid)", level: .warning)
         }
       }
+      if case .bool(let fileStatus) = options["sourcekit/fileStatus"] {
```

Comment:
Why an initialization option rather than a client capability? This feels more like a capability to me.